tests/merlin-wrapper-template

```diff
@@ -1,8 +1,8 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 src_dir="$(dirname ${BASH_SOURCE[0]})/.."
 cd "$src_dir"
-src_dir=$(realpath -e $(pwd))
+src_dir=$(realpath -e $(pwd) 2>/dev/null || readlink -f $(pwd))
```

Comment:
This change doesn't appear to be working correctly on my Trusty VM, but I haven't yet looked into why (equally, it's only a few more months before we don't have to worry about Ubuntu 14.04...)